test/integration/sensing.js

```diff
@@ -0,0 +1,35 @@
+var path = require('path');
+var test = require('tap').test;
+var extract = require('../fixtures/extract');
+var VirtualMachine = require('../../src/index');
+
+var uri = path.resolve(__dirname, '../fixtures/sensing.sb2');
+var project = extract(uri);
+
+test('sensing', function (t) {
```

Comment:
Totally agreed. I think it's probably something to follow-up with as a new fixture specifically for integration testing.